js/scripts.js

```diff
@@ -0,0 +1,17 @@
+var triangle1Area = getTriangleArea(7, 5);
+var triangle2Area = getTriangleArea(6, 9);
+var triangle3Area = getTriangleArea(3, 6);
+
+
+function getTriangleArea(a, h) {
+	if ( (a <= 0) || (h <= 0) ) {
+		 console.log("Incorrect data");
+	} else {
```

Comment:
you can write it simpler. just stop your function in previous if statement and leave return from next line without `else`